### PR TITLE
Release 7.0.0

### PR DESCRIPTION
**Pull request by crow**
release 7.0.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,25 @@
...
+- Updated Airship Android SDK to 13.2.1
+- Updated Airship iOS SDK to 13.4.0
+- Added channel tag editor.
+- Added named user and channel attribute editor.
+- Added named user and channel tag group editor.
+- Added iOS badge support.
+- Added screen tracking support.
+- Added flag to pause or resume the display of In-App Automation.
+- Added opt-in status flag.
+- Added enableUserNotifications method with a resulting status callback.
```
Comment (marc-scig) [inline CHANGELOG.md]: result status?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,25 @@
...
+- Added EUCS cloud site support.
+- Added more robust event handlers.
+- Added notifications for opt-in status changed and notification reponse events.
+- Added custom events support.
+- Added module version tracking.
```
Comment (marc-scig) [inline CHANGELOG.md]: probably not something we need to mention publicly

**Review by marc-scig — APPROVED** [review]
A few minor suggestions

**Review by BrianBatchelder — APPROVED** [review]
👍

**Pull request merged**